### PR TITLE
Landing page mobile

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  if (req.hostname.match(/storage\./g)) {
+    res.sendFile(path.resolve("index.html"));
+  } else {
+    next();
+  }
+};
+
+app.get("/storage*", subdomainMiddleware, (req, res) => {
+  res.sendFile(path.resolve("index.html"));
+});
```
Comment (EdmundMai) [inline index.js]: you don't want to commit this right?

File: package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "eslint-plugin-import": "2.8.0",
     "eslint-plugin-jsx-a11y": "5.1.1",
     "eslint-plugin-react": "7.4.0",
+    "express": "^4.16.4",
```
Comment (EdmundMai) [inline package.json]: this too

File: package.json [hunk 3]
```diff
@@ -127,7 +128,7 @@
     "tslint-react": "^3.6.0"
   },
   "scripts": {
-    "start": "cross-env NODE_ENV=development PORT=3001 webpack-dev-server --config='config/webpack.config.js' --mode development",
+    "start": "cross-env NODE_ENV=development PORT=3001 webpack-dev-server --config='config/webpack.config.js' --mode development && node ./index.js",
```
Comment (EdmundMai) [inline package.json]: and this?

File: index.js (hunk 1 above)
Comment (tenisakb) [reply index.js]: ah, of course not.. just removed

File: package.json (hunk 2 above)
Comment (tenisakb) [reply package.json]: removed

File: package.json (hunk 3 above)
Comment (tenisakb) [reply package.json]: reverted

**Pull request merged**